components/eam/src/physics/cam/check_energy.F90

```diff
@@ -21,6 +21,9 @@ module check_energy
 !                  MG2 (now includes prognostic rain and snow)    
 !               2. Better printout information for energy and water conservation check 
 !               3. Additional water conservation check utilities 
+!
```

Comment:
@oksanaguba ,These added comments and similar ones in clubbb_intr.F90 and physics_types.F90 seem to be intended when incorporating your other proposed energy fixes as well. Are we planning to add those at a later time? If not, I would think not to include the comments in these three files.

Reply:
those comments are for something that was merged in January. so the changes are already in. i was told to add comments like this, but i forgot to do it in January.

Reply:
I see now and that makes sense. Thanks for the explanation, @oksanaguba.